SwaprHQ/stackly-ui
# [SWA-212][REFACTOR] - Change FA implementation approach and add extra events

## Fixes: [SWA-212](https://linear.app/swaprhq/issue/STK-212/fix-fa-execution-when-blockers-are-running)

# Description
* Prune old FA implementation using loading scripts
* Prune not used `time` utils file
* Create a new component approach suggested by Next maintainers
* Added new events for Create Stack flow

# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Stackly Landing 
4) Connect your wallet
5) 
  a) On Sackly Landing, clicking any "Launch App" or "Stack Now" button should trigger a FA event 
  b) On Stackly dApp, clicking a Strategy Card should trigger a FA event
